.github/dependabot.yml

```
@@ -5,3 +5,9 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 10
+
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
```

I think part of the complaint before that there was too much noise, maybe this strikes a balance
```suggestion
    interval: weekly
```